addons/sourcemod/scripting/gokz-core/timer/virtual_buttons.sp

```diff
@@ -37,6 +37,11 @@ bool GetHasVirtualEndButton(int client)
 	return hasVirtualEndButton[client];
 }
 
+bool GetVirtualStartCourse(int client)
+{
+	return virtualStartCourse[client];
```

Comment:
This is an int, not a bool, right?

Reply:
whoops yeah. I'm not too fond of this PR though, because it kinda disables the behavior in #346 if there's a timer tech around, so consider this a temporary fix